Expand on tutorials for biologist focused understanding (#394)
cytomining/CytoTable: docs/source/tutorial.md

# Tutorials

Start here if you are new to CytoTable. We’ve split material by audience:

- **Image analysts (no engineering background required):** follow the narrative tutorials below. They include downloadable data, exact commands, and what to expect.
- **Engineers / power users:** see the Software Engineering Guide for tuning and integration details, or use the quick recipe below.

```{admonition} Who this helps (and doesn’t)

> - Helps: image analysts who want to get CellProfiler/NPZ outputs into Parquet with minimal coding; people comfortable running a few commands.

Reviewer: ```suggestion
- Helps: image analysts who want to get CellProfiler/DeepProfiler/InCarta outputs into Parquet with minimal coding; people comfortable running a few commands.
```

Consistent with what you have below. Not sure if you mention anywhere that DeepProfiler outputs as NPZ.

- Not ideal: raw image ingestion or pipeline authoring (use CellProfiler/DeepProfiler upstream); workflows needing a GUI-only experience.
- Effort: install, copy/paste a few commands, validate outputs in minutes.
```

```{toctree}
---
maxdepth: 2
caption: Tutorials (start here)
---
tutorials/cellprofiler_sqlite_to_parquet
tutorials/npz_embeddings_to_parquet

> tutorials/multi_plate_merge_tablenumber

Author: Added this into the cellprofiler tutorial with some branching based on env vars.

software_engineering
```

Looking for variations or troubleshooting? See the Software Engineering Guide.

## Quick recipe: CellProfiler CSV to Parquet

> This short recipe is for people comfortable with Python/CLI and parallels our older tutorial. If you prefer a guided, narrative walkthrough with downloadable inputs and expected outputs, use the tutorial above.

Reviewer: ```suggestion
This short recipe is for people comfortable with Python/CLI and parallels our older tutorial. 
If you prefer a guided, narrative walkthrough with downloadable inputs and expected outputs, use the tutorial above.
```
Recommend double-checking that there is one sentence per line.

Author: Thanks will do!

> [CellProfiler](https://cellprofiler.org/) exports compartment CSVs (for example, "Cells.csv", "Cytoplasm.csv"). CytoTable converts this data to Parquet from local or object-storage locations.

Reviewer: ```suggestion
[CellProfiler](https://cellprofiler.org/) exports compartment CSVs (for example, "Cells.csv", "Cytoplasm.csv"). 
CytoTable converts this data to Parquet from local or object-storage locations.
```
Same here, I won't get all examples but highly recommend doing one more pass before merge.
